complex.c

```diff
@@ -0,0 +1,128 @@
+#include <stdio.h>
+#include <math.h>
+
+typedef struct{
+	float aNum;
+	float bNum;
+
+
+}comp;
+
+float Asum(float a1, float a2){
+	int atotal;
```

Comment:
I think the fact that this is an int is a typo....you want to be careful of that!